content/en/docs/collector/custom-collector.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Building your own Collector instance
+weight: 30
+---
+
+If you are planning to build and debug custom collector receivers, processors,
+extensions, or exporters, you are going to need your own Collector instance.
+That will allow you to launch and debug your OpenTelemetry Collector components
+directly within your favorite Golang IDE.
+
+The other interesting aspect of approaching the component development this way
+is that you can use all the debugging features from your IDE (stack traces are
+great teachers!) to understand how the Collector itself interacts with your
+component code.
+
+The OpenTelemetry Community developed a tool called [OpenTelemetry Collector
+builder][ocb] (or `ocb` for short) to assist people in assembling their own
+distribution, making it easy to build a distribution that includes their custom
+components along with components that are publicly available.
+
+As part of the process the `builder` will generate the Collector's source code,
```

Comment:
Should `builder` be replaced with `ocb`

Reply:
I am torned, the OpenTelemetry Collector Builder documentation (https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder) still refers to the tool as builder. I would keep it.

Reply:
That works for me. @svrnm?